src/main/java/nl/dtls/fairdatapoint/api/config/RestApiContext.java

```diff
@@ -102,6 +106,13 @@ public void configureContentNegotiation(ContentNegotiationConfigurer configurer)
             converter.configureContentNegotiation(configurer);
         }
     }
+    
+    @Bean
+    public TaskExecutor threadPoolTaskExecutor() {
+        ThreadPoolTaskExecutor executor = new ThreadPoolTaskExecutor();

```

Comment:
You can create/reuse existing threadpools by using [`Executors.new*ThreadPool()`](https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/Executors.html#newCachedThreadPool--) (and pick one from the list that is appropriate: default, cached, fixed, scheduled, single, etc).